linera-execution/src/policy.rs

```diff
@@ -146,4 +146,20 @@ impl ResourceControlPolicy {
             ..Self::default()
         }
     }
+
+    /// Creates a policy that matches the Devnet launched in January 2024.
```

Comment:
I think we'd like this to match a (the most) recent devnet.

Reply:
It still does, until we use https://github.com/linera-io/linera-infra/pull/73.

But I guess it doesn't hurt to already add the updated fees? I'll add a commit.

Reply:
I had to add the approximate balance assertions. I think they do make sense in tests where we want to check whether a transfer of an integer token amount has arrived.

Reply:
But what's the alternative? E.g. in the faucet test, shouldn't we test that the faucet put the expected balance into the new chains?

Reply:
In this case, why is it approximate?

Reply:
You're right, those should be exact. For the others we can maybe use the owner account: https://github.com/linera-io/linera-protocol/issues/1637
I'll work on this tomorrow.